### PR TITLE
Use reps to render responses from the backend

### PR DESCRIPTION
**Pull request by linclark**
This PR takes everything from #94 except for the object linking. Once the object link patch lands, we can reintegrate that.

@bgrins, this is ready for review. @nchevobbe feel free to review if you like, too

### CODE REVIEW

File: devtools/client/webconsole/new-console-output/components/grip-message-body.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/* -*- indent-tabs-mode: nil; js-indent-level: 2 -*- */
+/* vim: set ft=javascript ts=2 et sw=2 tw=80: */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use strict";
+
+// React
+const {
```
Comment (bgrins) [inline devtools/client/webconsole/new-console-output/components/grip-message-body.js:10]: Why the new line around the PropTypes import here?

File: devtools/client/webconsole/new-console-output/test/components/test_console-api-call.html [hunk 2]
```diff
@@ -12,22 +12,47 @@
...
-  const { prepareMessage } = require("devtools/client/webconsole/new-console-output/utils/messages");
-  const { ConsoleApiCall } = require("devtools/client/webconsole/new-console-output/components/message-types/console-api-call");
+  yield testConsoleLog();
+  yield testConsoleCount();
+
+  SimpleTest.finish()
+});
 
+function testConsoleLog() {
   const packet = yield getPacket("console.log('foobar', 'test')", "consoleAPICall");
```
Comment (bgrins) [inline devtools/client/webconsole/new-console-output/test/components/test_console-api-call.html:26]: Should this packet and expected text be fetched from the testCommands map?  This seems like its purpose, and would get rid of duplicated command text and `// @TODO should output: foobar test` lines if we were fetching the "console.log()" command here

Comment (bgrins) [inline devtools/client/webconsole/new-console-output/test/components/test_console-api-call.html:26]: "Should this packet" -> "Should this command"

Comment (bgrins) [inline devtools/client/webconsole/new-console-output/test/components/test_console-api-call.html]: "Should this packet" -> "Should this command"

Comment (linclark) [inline devtools/client/webconsole/new-console-output/test/components/test_console-api-call.html:26]: In #98 the plan is to remove the `testCommands` map

File: devtools/client/webconsole/new-console-output/components/grip-message-body.js (hunk 1 above)
Comment (linclark) [inline devtools/client/webconsole/new-console-output/components/grip-message-body.js:10]: Once we reintegrate the object linking, we'll be importing `createFactory` here too.

**Comment by bgrins** [comment]
Nice and simple, fine with me

**Pull request merged**